File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/Report.java
package work.socialhub.kmastodon.entity;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * @author hecateball
 */
public class Report implements Serializable {

    @SerializedName("id")
    private String id;
    @SerializedName("actionTaken")
    private String actionTaken;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getActionTaken() {
        return actionTaken;
    }

    public void setActionTaken(String actionTaken) {
        this.actionTaken = actionTaken;
    }

}

File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/Report.kt
package work.socialhub.kmastodon.entity

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable
import kotlin.js.JsExport

@JsExport
@Serializable
class Report {

    @SerialName("id")
    var id: String? = null

    @SerialName("actionTaken")
    var actionTaken: String? = null
}
